tests/Parsers/Microsoft.Bot.Builder.Parsers.LU.Tests/Parser/LuParserTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+using Microsoft.Bot.Builder.Parsers.LU.Parser;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace Microsoft.Bot.Builder.Parsers.LU.Tests.Parser
+{
+    public class LuParserTests
+    {
+        [Theory]
+        [InlineData("LU_Sections")]
+        [InlineData("SectionsLU")]
+        [InlineData("ImportAllLu")]
+        public void ParseLuContent(string fileName)
+        {
+            // var luContent = "# Help"+ Environment.NewLine + "- help" + Environment.NewLine + "- I need help" + Environment.NewLine + "- please help";
+            var path = Path.Combine(Directory.GetCurrentDirectory(), "Fixtures", fileName + ".txt");
+            Console.WriteLine(path);
```

Comment:
>Console.WriteLine(path); [](start = 12, length = 24)

If you want to output this message, in xUnit use ITestOutputHelper instead of sending it to Console.